integration_test/step_1.py

```diff
@@ -0,0 +1,103 @@
+import substitution
+from general import *
+
+STORE_NAME = get_random_string(20)
+HOST = get_host()
+
+# clear data
+clear_data()
```

Comment:
why is clear data necessary?

Reply:
without executing this function and stopping the processes, the next step will fail